69-common-kingslayer/technical_specification.md

```diff
@@ -0,0 +1,51 @@
+# OpenTelemetry in file services (Common Kingslayer)
```

Comment:
Alternative title: "Distributive tracing with OpenTelemetry and Jaeger"

You can narrow the implementation scope to file services, but I think the main difficulty is to create spans that run across services, because that needs context propagation. Unless you actually want to keep this epic small and support only intra-service spans.